### PR TITLE
fix MIDI mapping wizard without existing mapping

### PR DESCRIPTION
**Pull request by Be-ing**
This is a work-in-progress with rough edges. I'd be happy if someone else finished this.

### CODE REVIEW

**Comment by Be-ing** [comment]
@Holzhaus are you interested in taking over here?

File: src/controllers/dlgprefcontroller.cpp [hunk 1]
```diff
@@ -42,6 +42,13 @@ DlgPrefController::DlgPrefController(QWidget* parent,
     // Create text color for the file and wiki links
     createLinkColor();
 
+    // Check the text color of the palette for whether to use dark or light icons
+    if (!Color::isDimColor(palette().text().color())) {
```
Comment (Holzhaus) [inline src/controllers/dlgprefcontroller.cpp:46]: There are some unrelared changes in this branch... I guess I'll start from scratch.

Comment (Be-ing) [reply src/controllers/dlgprefcontroller.cpp:46]: That was done for a reason. I don't remember exactly why, but I think you'll find out if you start over from scratch.

Comment (Holzhaus) [reply src/controllers/dlgprefcontroller.cpp:46]: ah ok, now I understand the rationale behind this.

**Pull request closed without merging**